### ISSUE TITLE
Casbin.NET Promotion Plan

### ISSUE THREAD

**Issue by sagilio** [casbin/Casbin.NET#201]
Casbin.NET has been very complete, and the v2 version will be released soon. I think it is a good time to introduce Casbin.NET to more people. 
Before Casbin.NET v2 is released, I need to consider the details of the plan. Any suggestions are welcome.

**Comment by Leonardo-Ferreira** [comment]
any news on this? still need help?

**Comment by hsluoyz** [comment]
@sagilio

**Comment by sagilio** [comment]
@Leonardo-Ferreira Yes, I may not have much time to deal with the promotion plan recently.

**Comment by hsluoyz** [comment]
@Leonardo-Ferreira do you have idea about how to proceed, can you help?

**Comment by Leonardo-Ferreira** [comment]
I think that the best way to promote is build a very strong case alongside a relevant adopter. This way, we can approach others and have sort of a "hey look how we solved that mess over there"

**Comment by hsluoyz** [comment]
@Leonardo-Ferreira can you help on it?
